Formula/qt.rb

```diff
@@ -24,39 +24,35 @@ class Qt < Formula
     sha256 "25c4a693c787860b090685ac5cbeea18128d4d6361eed5b1bfed1b16ff6e4494" => :mojave
   end
 
-  keg_only "Qt 5 has CMake issues when linked"
+  keg_only "Qt has CMake issues when linked"
```

Comment:
For the link issue... In my fork, I try to define these CMake variables
-DCMAKE_INSTALL_PREFIX=/usr/local
-DCMAKE_STAGING_PREFIX=#{prefix}

and define these variables to put cmake relate files to share folder
-DINSTALL_DESCRIPTIONSDIR=share/qt/modules
      -DINSTALL_DOCDIR=share/doc/qt
      -DINSTALL_EXAMPLESDIR=share/qt/examples
      -DINSTALL_MKSPECSDIR=share/qt/mkspecs
      -DINSTALL_PLUGINSDIR=share/qt/plugins
      -DINSTALL_QMLDIR=share/qt/qml
      -DINSTALL_TESTSDIR=share/qt/tests

I don't know this issue can be solved by these variables